test/e2e/vsn.js

```diff
@@ -0,0 +1,242 @@
+'use strict'
+
+const assert = require('assert')
+const tapePromise = require('tape-promise').default
+const tape = require('tape')
+const isRoughlyEqual = require('is-roughly-equal')
+
+const {createWhen} = require('./lib/util')
+const createClient = require('../..')
+const vsnProfile = require('../../p/vsn')
+const products = require('../../p/vsn/products')
+const {
+	line: createValidateLine,
+	journeyLeg: createValidateJourneyLeg,
+	movement: _validateMovement
+} = require('./lib/validators')
+const createValidate = require('./lib/validate-fptf-with')
+const testJourneysStationToStation = require('./lib/journeys-station-to-station')
+const testJourneysStationToAddress = require('./lib/journeys-station-to-address')
+const testJourneysStationToPoi = require('./lib/journeys-station-to-poi')
+const testEarlierLaterJourneys = require('./lib/earlier-later-journeys')
+const journeysFailsWithNoProduct = require('./lib/journeys-fails-with-no-product')
+const testDepartures = require('./lib/departures')
+const testArrivals = require('./lib/arrivals')
+
+const when = createWhen('Europe/Berlin', 'de-DE')
+
+const cfg = {
+	when,
+	products,
+	minLatitude: 47.24,
```

Comment:
Adapt these to a reasonably large bounding box around the VSN service area. The intention here is to catch bogus values (e.g. `{latitude: 0, longitude: 0}` for VSN), so make it rather large.